arch/arm/mach-exynos/exynos.c

```diff
@@ -206,7 +206,7 @@ DT_MACHINE_START(EXYNOS_DT, "SAMSUNG EXYNOS (Flattened Device Tree)")
 MACHINE_END
 
 #define ODROID_MACHINE_START(name, compat)			\
-	DT_MACHINE_START(EXYNOS5422_ODROID_##name, "ODROID-"#name)	\
+	DT_MACHINE_START(EXYNOS5422_ODROID_##name, "Hardkernel ODROID-"#name)	\
```

Comment:
If it's just for having a nice /proc/cpuinfo maybe leave it like on mainline?
Each extra patch away from mainline makes it harder to maintain :)

Reply:
Right, it is our traditional method for identifying a board.

AFAIK, our WiringPi and DietPi use /proc/cpuinfo to identify.